redisinsight/api/src/modules/database/database.analytics.spec.ts

```diff
@@ -263,10 +270,129 @@ describe('DatabaseAnalytics', () => {
             useSSH: 'disabled',
             verifyTLSCertificate: 'disabled',
             useTLSAuthClients: 'disabled',
+            forceStandalone: 'false',
           },
         },
       );
     });
+
+    it('should emit event with forceStandalone included', () => {
+      const prev = {
+        ...mockDatabase,
+      };
+      const cur = {
+        ...mockDatabase,
+        forceStandalone: true,
+      };
+      service.sendInstanceEditedEvent(mockSessionMetadata, prev, cur);
+
+      expect(sendEventSpy).toHaveBeenCalledWith(
+        mockSessionMetadata,
+        TelemetryEvents.RedisInstanceEditedByUser,
+        {
+          databaseId: cur.id,
+          connectionType: cur.connectionType,
+          provider: undefined,
+          useTLS: 'disabled',
+          verifyTLSCertificate: 'disabled',
+          useTLSAuthClients: 'disabled',
+          useSNI: 'disabled',
+          useSSH: 'disabled',
+          timeout: mockDatabaseWithTlsAuth.timeout / 1_000, // milliseconds to seconds
+          useDecompression: mockDatabaseWithTlsAuth.compressor,
+          forceStandalone: 'true',
+          previousValues: {
+            connectionType: prev.connectionType,
+            provider: prev.provider,
+            useTLS: 'disabled',
+            useSNI: 'disabled',
+            useSSH: 'disabled',
+            verifyTLSCertificate: 'disabled',
+            useTLSAuthClients: 'disabled',
+            forceStandalone: 'false',
+          },
+        },
+      );
+    });
+
+    it('should emit event with forceStandalone not included', () => {
+      const prev = {
+        ...mockDatabase,
+      };
+      const cur = {
+        ...mockDatabase,
```

Comment:
not sure tho, but 
```suggestion
        ...mockDatabase,
        forceStandalone: undefined,
```
to explicitly check it? In case someone modifies the `mockDatabase`. The point is to make the test robust